Example/Example Swift/Info.plist

```diff
@@ -49,7 +49,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.8.2</string>
+	<string>0.9.0</string>
```

Comment:
What's the reasoning behind the sample app versioning? Isn't it about time it got a version that's not 0.X?

Reply:
Just to identify that there is a new version for the beta, but it will be 1.0.0 when the new GVL version is released.

Reply:
Just don't forget it.